modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,66 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+ $(document).ready(function(){
+     // loading aria attribute on pates tabs
+     $('.nav-tabs').attr('role', 'tablist');         // ul
+     $('.nav-tabs li').attr('role', 'presentation'); // li
+     $('.nav-tabs a').attr('role', 'tab');           // a
+
+     // if no selected => select first
+     if ($('.nav-tabs a[aria-selected]').length == 0) {
+         $('.nav-tabs a:first').attr('aria-selected', 'true');
+         $('.nav-tabs a:first').attr('tabindex', '0');
+      }
+
+      /* click on a link */
+      $('.nav-tabs a').click(function() {
+          $this = $(this);
+
+          // remove aria selected on all links + remove focusable
+          $('.nav-tabs a').removeAttr('aria-selected');
+          $('.nav-tabs a').attr('tabindex', '-1');
```

Comment:
These need to be properly scoped, there can be multiple tab containers on a single page at any given moment.

Reply:
Any suggesting on this, how does October scope the tabs with the `class="active"` I could try to copy that one.

Reply:
@w20k Any suggests on this?

Reply:
Luke, I'm going to click 'Resolved' on this one. I've had a think about this and what you have said wouldn't apply to a keyboard control or screen reader. I give you an example:

- A web page has two sets of tabs. 
- October gives each tab set a `class="active"` (so we get two default tabs in total).
- The Screen Reader will ignore all of that and only select the the current tab - selected by the keyboard control.

Therefore, logically the Javascript is correct. Because the Javascript makes all the none-selected tabs `tabindex="-1"` and the selected tab (done by the keyboard control) `tabindex="0"`. 

So there is no conflict and works correctly as coded right now.

Hope that makes sense for you!

(The keyboard control selects the items one by one as you scroll from one item to the next, so two items are never selected at the same time).